### DISCUSSION TITLE
能否修改默认清晰度的方案呢？

### DISCUSSION THREAD

**Discussion by cauzp** [Richasy/Bili.Uwp#808] [想法]
现行的清晰度设置方案是最高清晰度或者最低清晰度，能否修改为优先720p，或者优先1080p，4k，然后没有对应清晰的话自动选择下一个清晰度的机制呢。

**Comment by cauzp** [comment]
@Richasy

**Reply by Richasy** [reply]
目前清晰度有记忆策略，在没有强制最高清晰度的情况下，当你调整了一个视频的清晰度后，它能够顺延到下一个视频。
事实上，目前B站的主流清晰度还是在1080P，这对绝大部分设备来说并不是一个特别高的清晰度。即便第一个视频打开就是4K，也可以再调整，不必一定优先720P或1080P定死
